logging/logging_test.go

```diff
@@ -306,6 +300,7 @@ func entryForTesting(payload interface{}) *logging.Entry {
 	}
 }
 
+// Call this function sparingly. It takes ~10s to get logs, even with indexed filters.
```

Comment:
Totally minor since this isn't exported, but I generally try to follow the GoDoc pattern and start the comments with the thing being described. Something like:
```suggestion
// allTestLogEntries should be called sparingly. It takes ~10s to get logs, even with indexed filters.
```